src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => {};
+import classNames from 'classnames';
+import { User } from '../types/User';
+
+type Props = {
+  user?: User | null;
```

Comment:
now you render this component only if a user exists, so you can remove null
```suggestion
  user: User;
```